dist/LICENSE

```diff
@@ -222,6 +222,7 @@ The text of each license is also included at licenses/LICENSE-[project].txt.
 	sirupsen (logrus) 1.4.2: https://github.com/sirupsen/logrus MIT
 	urfave (cli) 1.22.1: https://github.com/urfave/cli MIT
 	nsf (termbox-go) 0.0.0-20190817171036-93860e161317: https://github.com/nsf/termbox-go MIT
+	99designs (gqlgen) 0.11.1: https://github.com/99designs/gqlgen MIT
```

Comment:
Why is this added in the dist release? Is this something missed before?

Reply:
mistake